src/git-town-append

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+source "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/helpers/helpers.sh"
+
+
+function ensure_has_target_branch {
+  if [ -z "$target_branch_name" ]; then
+    echo_error_header
+    echo_error "No branch name provided."
+    exit_with_error newline
+  fi
+}
+
+
+function preconditions {
+  target_branch_name=$1
+  ensure_has_target_branch
+
+  parent_branch_name=$INITIAL_BRANCH_NAME
+  if [ "$parent_branch_name" != "$MAIN_BRANCH_NAME" ]; then
+    ensure_knows_parent_branches "$parent_branch_name"
+  fi
```

Comment:
based on the current implementation I don't think the if blockis necessary and we can just call
```
ensure_knows_parent_branches "$parent_branch_name"
```
and it will just return immediately if the parent branch is the main branch.